### ISSUE TITLE
[Category] Short description

### ISSUE THREAD

**Issue by peakaboolol128** [sp614x/optifine#6935]
## Description of Issue

Game crashes as soon as i launch minecraft

## Steps to Reproduce
1. What do you do... open minecraft
2. ...to find this bug? and it crashes

## OptiFine Version
 <html>    OptiFine HD U H8 pre5
<body>

-- 

</body>
</html>

## Installation Method

put the jar file in the mods folder
## Fabric/Forge Version
fabric-loader-0.14.8-1.19

## Other Installed Mods

replay mod

## Log Files/Crash Reports

The game crashed whilst initializing game
Error: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_778 failed

**Comment by Lordalls** [comment]
The problem is that fabric loader doesn't support optifine with optifabric when it is in pre-release

**Comment by hectorviov** [comment]
I'm getting this error too, fabric works fine with optifabric and optifine 1.19 pre 4, pre 5 and pre 6 has this error.

**Comment by EAZYBLACK** [comment]
thats an optifabric problem not optifine

**Comment by SachinSAgrawal** [comment]
Hello! I am having a similar issue. I have OptiFabric and OptiFine installed, but when launching with the ReplayMod, my game crashes with error code 1. I talked to a dev from ReplayMod, and they said that OptiFine is too poorly written that it does not allow the mixin from ReplayMod to be applied. They said that it was also not worth their time or effort trying to fix it, so I am posting here, hopping that the devs of OptiFine (or OptiFabric) do something about it, but until then, the ReplayMod will be getting a bad rep from me.

**Comment by Lolothepro** [comment]
> Hello! I am having a similar issue. I have OptiFabric and OptiFine installed, but when launching with the ReplayMod, my game crashes with error code 1. I talked to a dev from ReplayMod, and they said that OptiFine is too poorly written that it does not allow the mixin from ReplayMod to be applied. They said that it was also not worth their time or effort trying to fix it, so I am posting here, hopping that the devs of OptiFine (or OptiFabric) do something about it, but until then, the ReplayMod will be getting a bad rep from me.

ReplayMod is not compatible with OptiFabric

**Comment by SachinSAgrawal** [comment]
> 

Yeah, but I hope someone (the devs of OptiFine or OptiFabric) does something about it soon.
